README.md

```diff
@@ -35,7 +35,16 @@ npm i @ng-web-apis/intersection-observer
    use `waIntersectionThreshold` and `waIntersectionRootMargin` attributes to configure
    [IntersectionObserver options](https://developer.mozilla.org/en-US/docs/Web/API/IntersectionObserver/IntersectionObserver)
 
-    > **NOTE:** Keep in mind these are used one time in constructor so you cannot use binding, only strings. Pass comma separated numbers to set an array of thresholds..
+    > **NOTE:** Keep in mind these are used one time in constructor so you cannot use binding, only strings. Pass comma separated numbers to set an array of thresholds.
+
+### Usage with Jest
+
+DOM environment provided by Jest does not emulate IntersectionObserver api's and need to be mocked. You can add the following line to your `setup.ts`:
```

Comment:
Could you change `api's` to `API` please and add something like this to the end:
`to use mocks from [@ng-web-apis/universal](https://github.com/ng-web-apis/universal) package`